src/components/organisms/AddForm/Form.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+.form {
+    cursor: text;
+    background-color: $secondary-backgroundColor;
+    border-radius: 2.9rem;
+    &__post {
+        width: 95%;
+        height: 6rem;
+        textarea {
+            width: calc(100% - 4rem);
+            height: calc(100% - 4rem);
+            margin: 2rem;
+        }
+        svg {
+            margin: 1rem;
+            width: 4.5rem;
+        }
+    }
+    &__comment {
+        width: 40%;
+        height: 3.5rem;
+        textarea {
+            width: calc(100% - 1rem);
+            height: calc(100% - 1rem);
+            margin: 0.5rem;
+        }
+        svg {
+            margin: 0.7rem 1rem;
+        }
+    }
+    &__chat {
+        width: 80%;
+        height: 6rem;
+        textarea {
+            width: calc(100% - 3rem);
+            height: calc(100% - 3rem);
+            margin: 1.5rem;
+        }
+        svg {
+            margin: 0.7rem 1.3rem 1.3rem;
+            width: 4rem;
+        }
+    }
+    form {
+        width: 100%;
+        height: 100%;
+        display: flex;
+        justify-content: flex-start;
+        textarea {
+            outline: none;
+            font-size: 1.5rem;
+            font-family: $font-style;
+            resize: none;
+            color: $primary-color;
+            border: none;
+            background-color: $transparent;
```

Comment:
this variable is actually an overkill since we are not ussing any calculations for colors, in this case we have defined variable in css called 'transparent' :D

Reply:
Done